supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -164,6 +164,53 @@
 
 *See also*:
 
+[discrete]
+[[operator]]
+==== image:images/yes.png[yes] Operator (noun)
+*Description*: In the context of Kubernetes, an Operator is a method of packaging, deploying, and managing a
+Kubernetes application. A Kubernetes application is an application that is both
+deployed on a Kubernetes cluster (including OpenShift clusters) and managed
+using the Kubernetes APIs and `kubectl` or `oc` tooling.
+
+The term "Operator" in the context of Kubernetes is always capitalized to distinguish it from other types of operators, such as human or mathematical operators. For example:
```

Comment:
Maybe because you're using the **.Example** label, you don' need the "For example:" here?